itou/employee_record/management/commands/transfer_employee_records.py

```diff
@@ -148,6 +149,16 @@ def _parse_feedback_file(self, feedback_file, batch, dry_run):
 
             return 1
 
+        # Check for notification records :
+        # Notifications are not mixed with employee records
+        notification_number = 0
+        for record in records:
+            if record.get("typeMouvement") == "M":
+                notification_number += 1
+        if notification_number == len(records):
```

Comment:
ce  `if` sur le nombre de `records` est une manière maladroite je trouve de faire un `for...else`.

Reply:
Complètement d'accord, c'est en train d’être remanié dans une PR à suivre ou je simplifie et refactor pas mal de chose sur les management command F.S.